index.js

```diff
@@ -1 +1,120 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db');
+const server = express();
+server.use(express.json());
```

Comment:
right now there's only a small group of imports/server init steps/middleware, but as these scale upwards I recommend grouping them by those concerns and keeping them separated with an empty line/comment to keep your head wrapped around them properly. not necessary but something I found useful when first going through this content. and this is also just an optimization trick, but since `express` on line 4 is just a reference to what you captured in a variable on line 1, you can skip the middleman and just write `const server = require('express')()`. Not needed, just something interesting